### PR TITLE
Release Candidate 25.0.0

### PR DESCRIPTION
**Pull request by ShepJGoogle**
**Release Candidate 25.0.0**

### FeatureHighlight

* Added properties 'bodyColor' and 'titleColor' in MDCFeatureHighlightView
* Added 'nullable' nullability specifiers to 'innerHighlightColor', 'outerHighlightColor'
* Added properties 'bodyCOlor' and 'titleColor' in MDCFeatureHighlightViewController

### Palettes

* Remove 'redPalette','pinkPalette','purplePalette','deepPurplePalette','indigoPalette','bluePalette',
    'lightBluePalette','cyanPalette','tealPalette','greenPalette','lightGreenPalette','limePalette',
    'yellowPalette','amberPalette','orangePalette','deepOrangePalette','brownPalette','greyPalette',
    'blueGreyPalette' class methods from MDCPalette in favor of class properties

#### Changes

* [[Feature Highlight] Expose title and body color properties (#1496)](https://github.com/material-components/material-components-ios/commit/a43472d2497836d12b70782b65b17788cf05efd0) (Sam Morrison)

#### Changes

* [Removing palette (#1500)](https://github.com/material-components/material-components-ios/commit/f71cd14b5b3ec3cf123e327bba875faf8ebf8f56) (Martin Petrov)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+* Added properties 'bodyColor' and 'titleColor' in MDCFeatureHighlightView
+* Added 'nullable' nullability specifiers to 'innerHighlightColor', 'outerHighlightColor'
+* Added properties 'bodyCOlor' and 'titleColor' in MDCFeatureHighlightViewController
+
+### Palettes
+
+* Removed 'redPalette','pinkPalette','purplePalette','deepPurplePalette','indigoPalette','bluePalette',
+    'lightBluePalette','cyanPalette','tealPalette','greenPalette','lightGreenPalette','limePalette',
+    'yellowPalette','amberPalette','orangePalette','deepOrangePalette','brownPalette','greyPalette',
+    'blueGreyPalette' class methods from MDCPalette
```
Comment (randallli) [inline CHANGELOG.md]: this is more of a remove class methods in favor of class properties.

**Review by randallli — APPROVED** [review]
just a tiny nit on the change log api diff

File: CHANGELOG.md (hunk 1 above)
Comment (ShepJGoogle) [reply CHANGELOG.md]: Done

**Review by randallli — APPROVED** [review]
dont forget to copy changelog into the PR description.

**Pull request merged**